### PR TITLE
Adding fake_logger development utility

### PR DESCRIPTION
**Pull request by jan-zmeskal**
Pretty much everything is said in module's __doc__.

There is also new implementation of watching for local changes. However I also kept the old one just to be sure. The reason for that is that it really did not feel correct to use unix `tail -f` when python has its own native way for watching text streams.

### CODE REVIEW

**Comment by jan-zmeskal** [comment]
Original comment from @grafuls:
Although this functionality is going to become useful when testing, this doesn't provide any functionality for the tool itself, hence I don't think the utils dir is the best place for this. We can move this to a new 'test' dir but in that case I would rather have unit tests and use python's mock library.

Original comment from @ILpinto:
First its good idea for testing, maybe we can call it mock_logger, it needs to be separated from this project (disruption-generator), Or Like Gonzalo suggested in different directory and run it on demand.
I am ok with the implementation

Thank you for your comments soon guys, I'll get to them soon, right now I have to focus on some more urgent issue.

**Comment by jan-zmeskal** [comment]
Ok, so here's my take on this, @grafuls, @ILpinto :
- I don't think there's really a conflict between using this and mock library. When I was exploring ways how to listen to some local logs other than `tail -f`, I needed some development tool that would provide mock objects to be quickly at my disposal and this is the outcome. If we later implement something different for and more robust e.g. with mock library, I am completely okay with that. But this is what we got now. I am not saying it's perfect but it's definitely usable. I really suggest going for iterative approach here as opposed to aiming for perfect from the get-go. If we decide in the future that we want different implementation, we can always scrap this without any problem.
- Regarding where the fake_logger should live. I really don't care too much. If you want it to be in test directory, that's completely fine with me. The question is which test directory. Do we only want one in root directory as is the case now (`disruption_generator/tests`)? Or do we want one for all components (`disruption_generator/disruption_generator/tests`)? Or do we want one for each component (`disruption_generator/listener/tests`)?

**Comment by grafuls** [comment]
Let's use disruption_generator/tests/utils for now if we can agree on that

File: listener/utils/fake_logger.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    """Generate random string of letters and digits that is between 1 and max_length chars long."""
+    random_str = target_msg
+    while target_msg in random_str:  # Make sure target_msg in not included in generated string
+        random_chars = choices(string.ascii_letters + string.digits, k=randint(1, max_length))
+        random_str = ''.join(random_chars) + '\n'
+    return random_str
+
+
+def main():
+    parser = argparse.ArgumentParser(description=__doc__)
```
Comment (grafuls) [inline listener/utils/fake_logger.py]: Replace argparse for click. 
Example here:
https://github.com/grafuls/disruption_generator/commit/55be1a03382696bc0935cb7e09627ffa35e44294#diff-9a66eb29f78de6b36ed9340ebf6dd7f4R17

File: listener/utils/fake_logger.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    parser.add_argument('-l', '--log-file', action='store', required=True,
```
Comment (grafuls) [inline listener/utils/fake_logger.py]: You can have the script create the temp file for you with tempfile.NamedTemporaryFile

File: listener/utils/fake_logger.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
+"""
```
Comment (grafuls) [inline listener/utils/fake_logger.py]: We could also use loremipsum.get_sentece package which generates a random lorem ipsum sentence.

**Comment by grafuls** [comment]
How about something like this for a non-blocking fake logger?
https://github.com/grafuls/pywatch/blob/master/tests/utils.py

**Pull request merged**